pkgs/applications/networking/cluster/atmos/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "atmos";
+  version = "1.4.28";
+
+  src = fetchFromGitHub {
+    owner = "cloudposse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AqW//+x1tKzm5dJirXCLFuDBTk3dHHw7c6vwsVsI37U";
+  };
+
+  vendorSha256 = "sha256-bepv8Ns8vcpLpmO7fsRgGijB9B6kQ5HhxUhIuEpGtOY=";
+
+  ldflags = [ "-X github.com/cloudposse/atmos/cmd.Version=v${version}" ];
```

This will strip the executable.
```suggestion
  ldflags = [ "-s" "-w" "-X github.com/cloudposse/atmos/cmd.Version=v${version}" ];
```

---

This is different than the upstream Makefile but I have added this change

https://github.com/cloudposse/atmos/blob/994003239ba18aca9ac688f511dd0f13228f9529/Makefile#L21